rs/registry/canister/src/mutations/do_update_subnet.rs

```diff
@@ -250,27 +250,23 @@ impl Registry {
     }
 
     /// Validates that the SEV feature is not changed on an existing subnet.
+    ///
     /// Panics if the SEV feature is attempted to be changed.
     fn validate_update_sev_feature(&self, payload: &UpdateSubnetPayload) {
-        if payload.features.is_none() {
-            return;
-        }
```

Comment:
I assume that when features is None, we make no attempt to change features. If this is right, then keeping this would make sense. Maybe an even better option is this:

```
let Some(old_features) = &payload.features else {
    return;
};
```

Reply:
I assume that when features is None, we make no attempt to change

Regarding the code snippet: `payload` carries the new feature flag, not the old. 

Changed to 

```
        let Some(features) = payload.features else {
            return;
        };
```